OpenSim/Tools/AnalyzeTool.cpp

```diff
@@ -401,7 +401,15 @@ loadStatesFromFile(SimTK::State& s)
         _model->getSimbodyEngine().convertDegreesToRadians(*qStore);
         _model->getSimbodyEngine().convertDegreesToRadians(*uStore);
 
-        _statesStore = createStatesStorageFromCoordinatesAndSpeeds(*_model, *qStore, *uStore);
+        // used to use createStatesStorageFromCoordinatesAndSpeeds(*_model, *qStore, *uStore);
+        double ti = qStore->getFirstTime();
+        double tf = qStore->getLastTime();
+        uStore->addToRdStorage(*qStore, ti, tf);
+
+        delete _statesStore;
+        _statesStore = new Storage(512, "states");
+
+        _model->formStateStorage(*qStore, *_statesStore);
```

Comment:
The method loadStatesFromFile is used by the UI of the AnalyzeTool, it's hard to see what are the side-effects of this change without integrating. Also the plotter calls AnalyzeTool.setStatesStorage() and passes in a storage created on GUI side based on inputs from the plot dialog. Do you expect the behavior of that call to change? This has to be done because the plotter varies Coordinates internally and allows users to set Activation-override values,